src/commands/logStream/StreamListStep.ts

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep, IAzureQuickPickItem, nonNullValue } from "@microsoft/vscode-azext-utils";
+import { localize } from "../../utils/localize";
+import { IStreamLogsContext } from "./IStreamLogsContext";
+import { ILogStream, getActiveLogStreams } from "./logStreamRequest";
+
+export class StreamListStep extends AzureWizardPromptStep<IStreamLogsContext> {
+    public async prompt(context: IStreamLogsContext): Promise<void> {
+        const placeHolder: string = localize('selectStream', 'Select a stream');
+        const picks: IAzureQuickPickItem<ILogStream | undefined>[] = this.getPicks(context);
+        if (picks.length > 1) {
+            picks.push({ label: localize('stopAll', 'Stop all Streams'), data: undefined });
```

Comment:
lol.... nit: I'm not sure if it should just be `Stop all streams` since it's not exactly a title.